mandu/src/components/organisms/ProductWithOptionCard.js

```diff
@@ -0,0 +1,25 @@
+import CartOptionCard from "../molecules/CartOptionCard";
+import PaymentOptionCard from "../molecules/PaymentOptionCard";
+
+const ProductWithOptionCard = ({id, productName, carts, canControl = true}) => {
+    const imageUrl = process.env.REACT_APP_API_URL + "/images/" + id + ".jpg";
+
+    return (
+        <div className="cart-product-card">
+            <div className="flex items-center p-2">
+                <img src={imageUrl} alt={"상품 사진"} className="w-20 h-20 mr-2"/>
```

Comment:
```suggestion
                <img src={imageUrl} alt={productName} className="w-20 h-20 mr-2"/>
```
대체 텍스트에 맞게 img title이나 productName등으로 설정하면 더 시멘틱하고 접근성 좋은 코드가 될 것 같네요